common/source/docs/common-limited-firmware.rst

```diff
@@ -76,3 +76,19 @@ The missing features are listed below.
 
    -  Parachute
    -  Sprayer
+
+RAM Limitations
+===============
+
+Occasionally, in the above boards, RAM will become limited with enabled options in the firmware also. This is manifested by various symptoms:
+
+- MAVFtp does not work for fast connections to GCS
```

Comment:
```suggestion
- MAVFtp does not work: parameter fetches to GCS will be slow
```

Reply:
Should be `MAVFTP`  (FTP and MAV both being acronyms).

Reply:
done